index.html

```diff
@@ -195,10 +195,11 @@ <h1 class="title has-text-centered name-title">Hum Qing Ze</h1>
                         </div>
 
                         <div class="column is-4 has-text-centered" data-aos="fade-left">
-                            <h1 class="title has-text-centered name-title">Timothy Liu Kai Hui</h1>
+                            <h1 class="title has-text-centered name-title">Timothy Liu Kaihui</h1>
                             <p class="seperator-text">--role--</p>
```

Comment:
@tlkh You can add your role here, if you want. If no ideas then can just delete this line.